app/controllers/api/base_controller.rb

```diff
@@ -90,5 +91,14 @@ def render_error(status, type, data = {})
         }.merge(data)
       }, status: status
     end
+
+    def log_usage!
+      Thread.new do
+        log_time = Time.current.utc.strftime('%Y-%m-%d %H:%M')
+        redis = Redis.new(url: "#{Exercism.config.tooling_redis_url}/5")
+        redis.incr("#{Exercism.env}:request:url:#{request.path}:#{request.method}:#{log_time}")
+        redis.incr("#{Exercism.env}:request:user:#{current_user.handle}:#{log_time}") if user_signed_in?
```

Comment:
- Merge these two lines together.
- Move `if user_signed_in?` to `return unless user_signed_in?` at the start of the method.
- `request.path` probably needs escaping
- We should probably use `id` not `handle` for `current_user` (safer and smaller).